source/Ship.cpp

```diff
@@ -2665,16 +2665,18 @@ int Ship::Scan(const PlayerInfo &player)
 	// Check the target's outfit and cargo space. A larger ship takes longer to scan.
 	// Normalized around 200 tons of cargo/outfit space.
 	// A ship with less than 10 tons of outfit space or cargo space takes as long to
-	// scan as one with 10 tons. This avoids small sizes being scanned instantly, or
-	// causing a divide by zero error at sizes of 0.
+	// scan as one with 10 tons. This avoids small sizes being scanned instantly.
+	// If instantly scanning very small ships is desirable, this can be removed.
```

Comment:
I mean, the divide by zero comment is still true. A depth of 0 would result in a denominator of 0. I'd say better to keep the mention of dividing by zero so that if we do remove this lower limit, the comments at least call out that we need to watch for this.
```suggestion
	// scan as one with 10 tons. This avoids small sizes being scanned instantly, or
	// causing a divide by zero error at sizes of 0.
	// If instantly scanning very small ships is desirable, this can be removed.
```